pkg/pillar/types/diskmetrics.go

```diff
@@ -20,14 +20,28 @@ var ReportDiskPaths = []string{
 
 // ReportDirPaths  Report directory usage for these paths
 var ReportDirPaths = []string{
+	PersistConfigDir,
+	PersistStatusDir,
+	CertificateDirname,
+	SealedDirName,
+	ClearDirName,
+	PersistDebugDir,
+	PersistInstallerDir,
+	IngestedDirname,
+	SnapshotsDirname,
+	PersistCachePatchEnvelopes,
+	PersistCachePatchEnvelopesUsage,
```

Comment:
I'm wondering now if these dirs belong more to `AppPersistPaths`? I mean these three:
```
	SnapshotsDirname,
	PersistCachePatchEnvelopes,
	PersistCachePatchEnvelopesUsage,
```

Reply:
Maybe - since we have the downloader and verifier in the apps list.

Reply:
Moving them to AppPersistPaths.